### PR TITLE
reduce the amount of spelling issues / WORDLIST

### PR DESCRIPTION
**Pull request by m7pr**
The follow-up after https://github.com/insightsengineering/coredev-tasks/issues/235

A lot of cleaning.

Output of `spelling::spell_check_package()` is short

```{R}
  WORD              FOUND IN
cloneable         Callable.Rd:132
                  CallableCode.Rd:85
                  CallableFunction.Rd:147
                  CallablePythonCode.Rd:117
                  CDISCTealDataConnector.Rd:95
                  CDISCTealDataset.Rd:248
                  CDISCTealDatasetConnector.Rd:168
                  CodeClass.Rd:199
                  JoinKeys.Rd:258
                  MAETealDataset.Rd:244
                  PythonCodeClass.Rd:61
                  TealData.Rd:321
                  TealDataAbstract.Rd:376
                  TealDataConnection.Rd:550
                  TealDataConnector.Rd:423
                  TealDataset.Rd:751
                  TealDatasetConnector.Rd:638
Forkers           README.md:49,59,61
funder            teal.data-package.Rd:29
Github            README.md:37
Hoffmann          teal.data-package.Rd:29
iteratively       using-delayed-data-basic.Rmd:106
pre               cdisc_data.Rd:19
                  PythonCodeClass.Rd:45
                  teal_data.Rd:19
                  TealDataConnection.Rd:324
                  TealDataConnector.Rd:316
                  README.md:32
                  preprocessing-delayed-data.Rmd:14,16,55
                  testing-delayed-data.Rmd:133,137
                  using-delayed-data-advanced.Rmd:477
Pre               preprocessing-data.Rmd:2
                  preprocessing-delayed-data.Rmd:2
                  teal-data.Rmd:55,56
repo              README.md:9,15,57,61
reproducibility   cdisc_data.Rd:19
                  cdisc_dataset.Rd:37
                  cdisc_dataset_connector.Rd:50
                  CDISCTealDataset.Rd:128,184
                  CDISCTealDatasetConnector.Rd:108
                  dataset.Rd:56
                  dataset_connector.Rd:47
                  mae_dataset.Rd:30
                  MAETealDataset.Rd:108
                  mutate_data.Rd:30
                  mutate_dataset.Rd:57
                  python_dataset_connector.Rd:69
                  teal_data.Rd:19
                  TealData.Rd:111
                  TealDataAbstract.Rd:98,222,272,307,321,329
                  TealDataset.Rd:126,179
                  TealDatasetConnector.Rd:127,466
                  README.md:33
                  preprocessing-data.Rmd:51
                  preprocessing-delayed-data.Rmd:16
                  teal-data.Rmd:57
                  teal.data-with-python.Rmd:84
                  testing-delayed-data.Rmd:127
                  using-delayed-data-advanced.Rmd:141
Reproducibility   reproducibility.Rmd:2,12
                  teal.data-with-python.Rmd:82
ui                TealDataConnection.Rd:282
                  TealDatasetConnector.Rd:575
                  using-delayed-data-advanced.Rmd:233
UI                TealData.Rd:238
                  TealDataConnection.Rd:372,503
                  TealDataConnector.Rd:269
                  TealDatasetConnector.Rd:533,542,588
                  testing-delayed-data.Rmd:76
                  using-delayed-data-advanced.Rmd:27,262,336,423
```

### CODE REVIEW

File: inst/WORDLIST [hunk 1]
```diff
@@ -1,23 +1,13 @@
-ADSL
-ADTTE
-CDISC
-Dataname
 Forkers
 Github
```
Comment (pawelru) [inline inst/WORDLIST:2]: GitHub

File: inst/WORDLIST [hunk 2]
```diff
@@ -1,23 +1,13 @@
...
 UI
 cloneable
-dataname
-datanames
 funder
 iteratively
 pre
 repo
 reproducibility
 ui
```
Comment (pawelru) [inline inst/WORDLIST:13]: can we combine it with `"UI"` that we already have?

File: inst/WORDLIST (hunk 1 above)
Comment (m7pr) [reply inst/WORDLIST]: https://github.com/insightsengineering/teal.data/pull/150/commits/e6cc0826b7dc5ee70f77bb5dad17a0e521c8b19b

File: inst/WORDLIST (hunk 2 above)
Comment (m7pr) [reply inst/WORDLIST]: https://github.com/insightsengineering/teal.data/pull/150/commits/e6cc0826b7dc5ee70f77bb5dad17a0e521c8b19b

**Review by pawelru — APPROVED** [review]

**Pull request merged**